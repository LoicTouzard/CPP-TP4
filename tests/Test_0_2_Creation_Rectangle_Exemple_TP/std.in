R rectangle1 56 46 108 4536
LIST
EXIT
