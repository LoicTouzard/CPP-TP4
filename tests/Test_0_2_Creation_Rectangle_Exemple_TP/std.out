OK
#New object: rectangle1
